docs-mslearn/fabric/create-finops-workspace.md

```diff
@@ -0,0 +1,235 @@
+description: This article walks you through creating and configuring a Microsoft Fabric workspace for FinOps.
+author: bandersmsft
+ms.author: banders
+ms.date: 07/17/2024
+ms.topic: how-to
+ms.service: finops
+ms.reviewer: micflan
+---
+
+# Create a Fabric workspace for FinOps
+
+This article walks you through creating and configuring a [Microsoft Fabric](/fabric/get-started/microsoft-fabric-overview) workspace for FinOps. It provides a step-by-step guide to create and configure Azure Data Lake storage (ADLS) setup, data export, workspace creation, and data ingestion. When completed, you can use Power BI to build reports.
+
+## Prerequisites
+
+Before you begin, you must have:
+
+- A paid Azure subscription with cost and usage information
+
+To complete this walkthrough, you create the following resources that incur costs:
+
+- An Azure Data Lake Storage Gen2 account
+- A Fabric or Power BI workspace
+
+## Create and configure Azure Data Lake Storage
+
+Microsoft Fabric is optimized to work with storage accounts with hierarchical namespace enabled, also known as Azure Data Lake Storage Gen2. You have two options to get a Data Lake storage account:
+
+- Option 1: To create a Data Lake storage account, see [Create a storage account for Azure Data Lake Storage Gen2](/azure/storage/blobs/create-data-lake-storage-account).
+- Option 2: To enable hierarchical namespace on an existing storage account, see [Upgrade Azure Blob Storage with Azure Data Lake Storage Gen2 capabilities](/azure/storage/blobs/upgrade-to-data-lake-storage-gen2-how-to?tabs=azure-portal).
+
+>[!TIP]
+>If you deployed [FinOps hubs](https://aka.ms/finops/hubs), you can use the storage account created as part of the hub resource group.
+
+For the following example, we used the following Data Lake Gen 2 storage account. You use your own storage account and subscription details.
+
+- Subscription = *Contoso subscription*.
+- Storage account = *Contoso-storage-account*.
+
+:::image type="content" source="./media/create-fabric-workspace-finops/storage-account-configuration-gen-2.png" border="true" alt-text="Screenshot showing converting an existing storage account into Data Lake storage." lightbox="./media/create-fabric-workspace-finops/storage-account-configuration-gen-2.png" :::
+
+## Export cost data
+
+You can use existing actual or amortized cost exports; however, the following example creates a new cost and usage details (FOCUS) data export.
+
+What is FOCUS? FinOps Open Cost and Usage Specification (FOCUS) is a new, cross-cloud billing data standard hosted by the FinOps Foundation and defined by Microsoft and a collection of FinOps practitioners, cloud providers, and vendors. To learn more about FOCUS and its benefits, see [About FOCUS](https://microsoft.github.io/finops-toolkit/focus). The FinOps Foundation also offers a free [Introduction to FOCUS course](https://learn.finops.org/introduction-to-focus). Microsoft’s Power BI solutions for FinOps are aligned to FOCUS.
```

Comment:
I want to lean on FOCUS a little more to encourage people to use that rather than actual/amortized.
```suggestion
Since we're creating a workspace optimized for FinOps, we'll export cost data using the FinOps Open Cost and Usage Specification (FOCUS), a provider-agnostic data format for cost details. All our examples will use FOCUS but you can also use existing actual or amortized cost exports that have file overwriting enabled.

To learn more about FOCUS and its benefits, see [About FOCUS](https://microsoft.github.io/finops-toolkit/focus). The FinOps Foundation also offers a free [Introduction to FOCUS course](https://learn.finops.org/introduction-to-focus). Microsoft’s Power BI solutions for FinOps are aligned to FOCUS.
```